flight/Libraries/math/cic.c

```diff
@@ -0,0 +1,160 @@
+/**
+ ******************************************************************************
+ * @addtogroup TauLabsLibraries Tau Labs Libraries
+ * @{
+ * @addtogroup TauLabsMath Tau Labs math support libraries
+ * @{
+ *
+ * @file       pid.c
+ * @author     The OpenPilot Team, http://www.openpilot.org Copyright (C) 2010.
```

Comment:
I am editing with eclipse, and these parts were collapsed by default, and I did not noticed them.
But will be fixed